src/pages/paintings.js

```diff
@@ -1,8 +1,94 @@
 import * as React from "react"
+import { Composition, Box } from "atomic-layout"
+import styled from "styled-components"
 import { DefaultLayout } from "../components/DefaultLayout"
 
+import Section from "../components/Section"
+import { Button } from "../components/Button"
+import one from "../images/one.png"
+import two from "../images/two.png"
+import three from "../images/three.png"
+import four from "../images/four.png"
+
+const StyledBox = styled.div`
+  text-transform: uppercase;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  align-items: center;
+  margin-top: var(--spacing-20);
+`
+
+const StyledTitle = styled.h3`
+  color: var(--color-white);
```

Comment:
Note that it's not the heading that's white, but the _Section_ component makes all its children have white text. This heading, just as any other text element, will inherit the text color by default.

TL;RD: Try to avoid setting colors on headings and paragraphs—they work magically when inheriting the color (which they do by default).

```diff
h3 {
  # Not all headings are white!
-  color: var(--color-white);
}

SectionStyles {
  // But all text _inside_ the black section is white.
  color: var(--color-white)
}
```